gui/src/app/SPAnalysis/SPAnalysisDataModel.ts

```diff
@@ -9,7 +9,7 @@ type SPAnalysisFiles = {
     [filetype in SPAnalysisKnownFiles]: string
 }
 
-type SPAnalysisBase = SPAnalysisFiles &
+export type SPAnalysisBase = SPAnalysisFiles &
```

Comment:
Are we actually using the fact that this is exported somewhere?

Reply:
I don't think so--probably an artifact.